### PR TITLE
Pass access token onto API

### PR DESCRIPTION
**Pull request by markysoft**
https://eaflood.atlassian.net/browse/PSD-687

The access token from Okta for the custom scope should be passed on to
the payment services API

### CODE REVIEW

File: app/plugins/auth.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 const bell = require('@hapi/bell')
 const authCookie = require('@hapi/cookie')
-const config = require('../config')
 
+const config = require('../config')
 const isSecure = config.isProd
 const redirectTo = config.oktaEnabled ? '/auth/okta' : '/auth/dev'
 
+const oktaCustomProvider = require('./okta-custom-provider')
+bell.providers['okta-custom'] = oktaCustomProvider
```
Comment (neilbmclaughlin) [inline app/plugins/auth.js:10]: Can these two lines be combined to remove `oktaCustomProvider`?

File: app/plugins/okta-custom-provider.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// this is work done by Tim Costa (https://github.com/timcosta)
```
Comment (neilbmclaughlin) [inline app/plugins/okta-custom-provider.js:1]: I think it's worth either tidying up this or wrapping it in comments which make it explicit that this is a cut and paste from the PR and doesn't conform to our standards. If it's not a cut and paste then we should change it to use `const` and string interpolation.

File: app/plugins/okta-custom-provider.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+const Joi = require('@hapi/joi')
+
+const internals = {
+  schema: Joi.object({
+    uri: Joi.string().uri().required(),
+    authorizationServerId: Joi.string().alphanum()
+  }).required()
+}
+
+exports = module.exports = function (options) {
```
Comment (neilbmclaughlin) [inline app/plugins/okta-custom-provider.js:14]: Not sure why the okta custom provider uses this form for module exports. I can see it's a multiple assign but I don't know what it achieves here.

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-payment-web",
   "description": "Digital service mock to  view scheduled payments for financial aid in the event a property subsides into mine shaft.",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (neilbmclaughlin) [inline package.json:4]: Not sure how, as a team, we decide what is a patch increment and what is a minor increment. I'd have chosen minor as it's not a bug fix and is adding functionality.

Comment (markysoft) [reply package.json]: thanks, all good points. I've made the suggested changes. I've updated the code from the Bell PR to imrpove the use of string templates and remove the let.

**Pull request merged**